docs/index.md

```diff
@@ -0,0 +1,24 @@
+Hyperbahn
+=========
+
+Hyperbahn is a service discovery and routing network that allows simple, reliable
+communication with any service without any knowledge of where that service runs.
+
+Hyperbahn supports the TChannel RPC protocol and works with Ringpop's gossip protocol
+and consistent hash ring topology.
+
+Hyperbahn and TChannel clients are currently supported in Go, Python, and Node.js.
+
+* Go
+    * [Guide](go-guide.md)
+    * [API Documentation](http://godoc.org/github.com/uber/hyperbahn-go)
```

Comment:
This link should probably be:
https://godoc.org/github.com/uber/tchannel-go/hyperbahn
or
https://godoc.org/github.com/uber/tchannel-go